policyengine_us/parameters/gov/states/hi/tax/income/credits/low_income_household_renters/base_tax_credit.yaml

```diff
@@ -0,0 +1,12 @@
+description: Hawaii base income tax credit for low-income household renters.
+values:
+  2022-01-01: 50
```

Comment:
we need to check the 2021 tax form to see if it is the same value and add the 2021 start date

Reply:
working on it.